shardingsphere-distribution/shardingsphere-proxy-distribution/pom.xml

```diff
@@ -45,7 +45,13 @@
             <artifactId>postgresql</artifactId>
             <scope>runtime</scope>
         </dependency>
-        
+
+        <dependency>
+            <groupId>org.opengauss</groupId>
+            <artifactId>opengauss-jdbc</artifactId>
+            <scope>runtime</scope>
```

Comment:
Could we use `provided` but not `runtime`? Just like mysql-connector.

Reply:
openGauss is open and free database, just like postgresql database, so my config is flowing pg.